### PR TITLE
Variable price shipping costs #31 issue/31

### PR DESCRIPTION
**Pull request by cklosowski**
#31

### CODE REVIEW

File: edd-simple-shipping.php [hunk 1]
```diff
@@ -222,8 +223,65 @@ protected function item_has_shipping( $item_id = 0, $price_id = 0 ) {
...
+	 */
+	public function get_price_shipping_cost( $download_id = 0, $price_id = null, $region = 'domestic' ) {
+		$download = new EDD_Download( $download_id );
+		$amount   = 0;
+		if ( $download->has_variable_prices() ) {
+			$prices = $download->get_prices();
+			foreach ( $prices as $key => $price ) {
+
+				// If it's not the right price ID, move along.
+				if ( (int) $key !== (int) $price_id ) { continue; }
```
Comment (pippinsplugins) [inline edd-simple-shipping.php]: Let's separate this into three lines please.

File: edd-simple-shipping.php [hunk 2]
```diff
@@ -222,8 +223,65 @@ protected function item_has_shipping( $item_id = 0, $price_id = 0 ) {
...
+
+				if ( isset( $price['shipping'] ) && is_array( $price['shipping'] ) ) {
+					// If the region requested isn't set, continue;
+					if ( ! isset( $price['shipping'][ $region ] ) ) { continue; }
```
Comment (pippinsplugins) [inline edd-simple-shipping.php]: 3 lines please :)

File: includes/admin/metabox.php [hunk 3]
```diff
@@ -22,13 +22,46 @@ public function __construct() {
...
+						} else {
+							$('#edd_simple_shipping_fields,.edd_prices_shipping').hide();
+						}
+					});
+
+					$('#edd_variable_pricing').on('click', function() {
+						var enabled  = $(this).is(':checked');
+						var shipping = $('#edd_enable_shipping').is(':checked');
+
+						if ( ! shipping ) { return; }
```
Comment (pippinsplugins) [inline includes/admin/metabox.php]: 3 lines please

**Comment by cklosowski** [comment]
@pippinsplugins Fixed.

**Pull request merged**